### ISSUE TITLE
[Bug] Starting print function makes whole Firefox flickering forever (constant switch between like and dark)

### ISSUE THREAD

**Issue by bcutter** [darkreader/darkreader#10869]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this bug to avoid opening a duplicate.
- [X] I can reproduce this bug in a new, unmodified web browser profile with Dark Reader installed as the only extension.
- [X] I understand I need to use the [Broken Website Report template](https://github.com/darkreader/darkreader/issues/new?assignees=&labels=Broken+Website&template=broken-website-report.yml&title=%5BBroken+Website%5D+Replace+with+title) if this bug I am reporting occurs on a single website.

### Bug Description

Doesn't matter on which website - once I open the print dialogue (CTRL + P) when dark mode is active, the print dialogue pops up and immediately the whole Firefox starts to flicker by constantly switching between the light and dark mode. Sometimes this makes the browser unresponsive, needing to force quit it.

### Website Address

https://website-does-not-matter.com

### Steps To Reproduce

1. Enable dark mode (is enabled by automation, following system theme)
2. Try to print any website

### Expected Behavior

Print dialogue comes up, no effect on light/dark mode.

### Actual Behavior

Flickering as described above.

### Operating System

Windows 10

### Web Browser name and version

Firefox 110.0.1

### Dark Reader version

4.9.62

### Additional Context

Seeing this on all machines (same browser, same OS, same set of other addons etc.).
There is no other extension/addon taking control of light/dark theming.

**Comment by robpneu** [comment]
This has been driving me crazy as well as I accidentally hit print every so often. I'll add that this is also happening on macOS. If you have enough tabs open it effectively locks up Firefox and I have to force quit it.

**Comment by bcutter** [comment]
Yes it's a major issue. One that no one cares unfortunately. Maybe people don't print anymore these days so only few run into this?
